My project/Assets/Scripts/GameLogic/TimeManager.cs

```diff
@@ -7,20 +7,50 @@ public class TimeManager : MonoBehaviour
 {
     public static TimeManager Instance {get; private set;}
     private float Timer = 0;
-    private float SecondsToDay = 60f;
+
+    //amount of seconds
+    [SerializeField] private float secondsToDay = 60f;
+
+    //amount of seconds for daytime
+    [SerializeField] private float daytimeSeconds = 45f;
+
+    //amount of days that have pased
+    public int Day { get; private set; }
+
+    
+    public bool IsDay { get; private set; }
     // Start is called before the first frame update
     void Start()
     {
-        
+        IsDay = true;
     }
 
     // Update is called once per frame
     void FixedUpdate()
     {
-        if(Timer < SecondsToDay){
-            Timer += Time.deltaTime;  
+        //Debug.Log(Timer);
+        if(Timer < secondsToDay){
+            Timer += Time.deltaTime;
+            if(Timer > daytimeSeconds && IsDay)
+            {
+                ChangeToNight();
+            }
         } else {
-            
+            Timer = 0;
+            Day++;
+            ChangeToDay();
         }
     }
+
+    private void ChangeToNight()
+    {
+        Debug.Log("Change to night time");
+        IsDay = false;
+    }
+
+    private void ChangeToDay()
+    {
+        Debug.Log("ChangeToDay to day time");
+        IsDay = true;
+    }
 }
```

Comment:
would it be easier to have isDay and Days be a derived field based on the elapsed time? for example:

`
    void FixedUpdate()
    {
        Timer += Time.deltaTime;
    }

public int Days () {
   return int(Timer) / int(secondsToDay)
}

public bool isDay() {
    return (int(Timer) % secondsToDay)) < daytimeSeconds;
}
`